src/test/java/seedu/address/testutil/PartnerCourseUtil.java

```diff
@@ -12,40 +12,35 @@ public class PartnerCourseUtil {
      * Returns an add command string for adding the {@code PartnerCourse}.
      */
     public static String getPartnerCourseAddCommandFrom(PartnerCourse partnerCourse) {
-        return PartnerCourseAddCommand.COMMAND_WORD
-                + " "
-                + PartnerCourseAddCommand.ACTION_WORD
-                + " "
-                + getPartnerCourseArgumentsForAddCommand(partnerCourse);
+        return String.format("%s %s %s",
+            PartnerCourseAddCommand.COMMAND_WORD,
+            PartnerCourseAddCommand.ACTION_WORD,
+            getPartnerCourseArgumentsForAddCommand(partnerCourse));
     }
 
     /**
      * Returns the part of add command string for the given {@code PartnerCourse}'s details.
      */
     public static String getPartnerCourseArgumentsForAddCommand(PartnerCourse partnerCourse) {
-        StringBuilder sb = new StringBuilder("[");
-        sb.append(partnerCourse.getPartnerCode()).append("] [");
-        sb.append(partnerCourse.getPartnerName()).append("]");
-        return sb.toString();
+        return String.format("[%s] [%s]",
+            partnerCourse.getPartnerCode(),
+            partnerCourse.getPartnerName());
```

Comment:
I think there should be PartnerUnit as well!

Reply:
wait and university haha.. my bad for not putting that earlier

Reply:
done